conf/modules.config

```diff
@@ -44,6 +44,56 @@ process {
         ext.prefix = { "${meta.id}.fa" }
     }
 
+    withName: MMSEQS_DATABASES {
+        publishDir = [
+            path: { "${params.outdir}/databases/mmseqs/" }, // dir==mmseqs_database/
```

Comment:
Is the `// dir==mmseqs_database/` necessary? I don't know what it wants to tell me.